auth-server/src/resolvers/mutation/changeUsername.ts

```diff
@@ -10,6 +12,10 @@ interface argsType {
 export default async (parent, { username }: argsType, ctx: Context) => {
 	let token = getTokenFromReq(ctx.req)
 
+	if (username.length < 3) {
```

Comment:
I have added this check in https://github.com/paritytech/polkassembly/pull/152 also. You will get merge conflict